src/services/EventService.ts

```diff
@@ -21,6 +21,14 @@ export default class EventService {
       const snoozedEvents = await SnoozedEventModel.find();
 
       responseData.events.forEach((event: any, index: number, events: any[]) => {
+        const messageElements = event.state.message.split('@@@');
+
+        if (messageElements.length === 3) {
```

Comment:
Please add error handling, in case the split string does not contain 3 elements (else case).

Reply:
Good point 👍 Please take another look